### PR TITLE
Fix #260 social links in footer will open in new tab

### LINKED ISSUE #260: Open the footer social link in new tab
### Describe the bug
Right now the link of social icons opens in the same tab, it will be better if it opens in another tab so that the user won't get redirected from the main website.

### To Reproduce
1. Go to 'https://anitab-org.github.io/'
2. Scroll down to 'footer'
3. Click on 'any of the social icon'
4. See error 'as it opens in the same tab'

### Expected behavior
When we click on the social icon in the footer the link should open in a new tab

### Desktop (please complete the following information):
 - OS: [ubuntu 18.04]
 - Browser [ chrome]

### PR DESCRIPTION
**Pull request by darshanhande11**
### Description
The social links in the footer open in a new tab on clicking them.

Fixes #260 

### Type of Change:
- Code

**Code/Quality Assurance Only**
- Bug fix #260 

### How Has This Been Tested?
By running the site on localhost, I have verified that the social handles links in the footer will open in a new tab.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by darshanhande11** [comment]
@keshakaneria Please review my PR and let me know if there are any changes to be done.

**Review by keshakaneria — APPROVED** [review]
Looks good 😄

**Review by nlok5923 — APPROVED** [review]
LGTM !!

File: yarn.lock [hunk 1]
```diff
@@ -405,6 +405,15 @@
     "@babel/traverse" "^7.9.0"
     "@babel/types" "^7.9.0"
 
+"@babel/highlight@^7.10.4":
```
Comment (codesankalp) [inline yarn.lock:408]: Is this file change required @darshanhande11?

File: src/Components/Footer/Social.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ function Social() {
       <Text style={[styles.text, styles.heading]}>Follow us here</Text>
       <View style={styles.socialContainer}>
         <TouchableHighlight
-          onPress={() => Linking.openURL('https://twitter.com/AnitaB_org')}
+          onPress={() => window.open('https://twitter.com/AnitaB_org','_blank')}
```
Comment (codesankalp) [inline src/Components/Footer/Social.js:11]:
```suggestion
          onPress = {() => window.open('https://twitter.com/AnitaB_org', '_blank')}
```
Same changes for other lines.

**Review by codesankalp — CHANGES_REQUESTED** [review]
I have reviewed this @darshanhande11. The changes are working fine. Please look into my comments and do the changes.

File: yarn.lock (hunk 1 above)
Comment (darshanhande11) [reply yarn.lock:408]: @codesankalp These changes occurred in yarn.lock file after I ran `yarn`. So, I didn't do those changes voluntarily.

File: src/Components/Footer/Social.js (hunk 2 above)
Comment (darshanhande11) [reply src/Components/Footer/Social.js]: @codesankalp I have done the suggested changes.

File: yarn.lock (hunk 1 above)
Comment (codesankalp) [reply yarn.lock:408]: Please unstage this file as it is not required here.

Comment (darshanhande11) [reply yarn.lock]: @codesankalp I have removed the unrequired code from yarn.lock

**Review by codesankalp — APPROVED** [review]
Good work @darshanhande11 :+1: 

Code-Review: Done
Tested locally:
Screenshot/gif:
b-2021-04-18_18 52 44

Expected result: All social links in the footer must open in a new tab.
Actual result: All social links are opening in a new tab.

**Pull request merged**